src/ripple/protocol/Feature.h

```diff
@@ -74,7 +74,7 @@ namespace detail {
 // Feature.cpp. Because it's only used to reserve storage, and determine how
 // large to make the FeatureBitset, it MAY be larger. It MUST NOT be less than
 // the actual number of amendments. A LogicError on startup will verify this.
-static constexpr std::size_t numFeatures = 53;
+static constexpr std::size_t numFeatures = 54;
```

Comment:
We should consider ways to remove this altogether. It's a bit of a workaround.

Reply:
Does this need to be updated to 55? Was another amendment added while this was being reviewed? CI fails on this number being wrong.